mlir/test/Dialect/Ptr/canonicalize.mlir

```diff
@@ -0,0 +1,15 @@
+// RUN: mlir-opt --canonicalize %s | FileCheck %s
+
+/// Check `ptr_add` canonicalizer patterns.
+
+// CHECK-LABEL: @ops0
+func.func @ops0(%ptr: !ptr.ptr<#ptr.generic_space>) -> !ptr.ptr<#ptr.generic_space> {
+  // CHECK: (%[[PTR_0:.*]]: !ptr.ptr<#ptr.generic_space>)
```

Comment:
```suggestion
// CHECK-LABEL: @zero_offset
// CHECK-SAME: (%[[PTR_0:.*]]: !ptr.ptr<#ptr.generic_space>)
func.func @zero_offset(%ptr: !ptr.ptr<#ptr.generic_space>) -> !ptr.ptr<#ptr.generic_space> {
```
ultra nit: I would use a more descriptive test name (and I like to move the argument check-line above the function.